configs/minimal.yaml

```diff
@@ -156,11 +156,13 @@ WHISK_PROPOSER_SELECTION_GAP: 1
 # EIP7594
 NUMBER_OF_COLUMNS: 128
 MAX_CELLS_IN_EXTENDED_MATRIX: 768
-DATA_COLUMN_SIDECAR_SUBNET_COUNT: 32
+DATA_COLUMN_SIDECAR_SUBNET_COUNT: 128
```

Comment:
Seems like making each subnet hold a single column allows simplifications in `get_custody_columns()` and `compute_subnet_for_data_column_sidecar()`. Basically wherever `DATA_COLUMN_SIDECAR_SUBNET_COUNT` is used.

Is this property something we don't want to enshrine? Or do we consider the simplifications out-of-scope for this PR?

Reply:
I think we will lose configurability. If in the future we think that 128 is too high for the number of subnets, it will be hard to reduce easily.

Reply:
Or we might want to increase the number of columns to make sampling cheaper